docs/pkgdown.css

```diff
@@ -62,7 +62,7 @@ img.icon {
   height: auto;
 }
 
-/* Fix bug in bootstrap (only seen in firefox) */
+/* Fix bug in bootstrap (only seen in Firefox) */
```

Comment:
brand.

(If this comes from somewhere, I could eventually send a PR to that...)

Reply:
Thanks, this is from pkgdown, here: https://github.com/r-lib/pkgdown